feather/plugin-host/src/host_calls/plugin_message.rs

```diff
@@ -0,0 +1,24 @@
+use feather_common::events::PluginMessageEvent;
+use feather_plugin_host_macros::host_function;
+
+use crate::context::{PluginContext, PluginPtr};
+
+#[host_function]
+pub fn plugin_message_send(
+    cx: &PluginContext,
+    channel_ptr: PluginPtr<u8>,
+    channel_len: u32,
+    data_ptr: PluginPtr<u8>,
+    data_len: u32,
+) -> anyhow::Result<()> {
+    let channel = cx.read_string(channel_ptr, channel_len)?;
+    let data = cx.read_string(data_ptr, data_len)?.into();
```

Comment:
This will fail when the data isn't valid UTF-8, so we need to get a `Vec<u8>` directly rather than going through `String`.

Reply:
Should I add a `read_bytes` method?

Reply:
Yes, that would make sense.